packages/commonwealth/client/scripts/state.ts

```diff
@@ -31,6 +29,7 @@ import type { MobileMenuName } from './views/app_mobile_menus';
 import type { SidebarMenuName } from './views/components/sidebar';
 import $ from 'jquery';
 import { updateActiveUser } from 'controllers/app/login';
+import { ChainCategoryType } from '../../../common-common/src/types';
```

Comment:
Can we use an absolute rather than relative import here?

Reply:
✅